File: src/org/lwjgl/LWJGLUtil.java
package org.lwjgl;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class LWJGLUtil {
    private static Logger LOGGER = LogManager.getLogger(LWJGLUtil.class);
    public static void log(CharSequence msg){
        LOGGER.info(msg.toString());
    }
    public static void log(String msg){
        LOGGER.info(msg);
    }
}

File: src/org/lwjgl/LWJGLUtil.kt
package org.lwjgl

import org.apache.logging.log4j.LogManager.getLogger
import org.apache.logging.log4j.Logger

/**
 * overriding the logger for LWJGL
 */
@Suppress("unused")
object LWJGLUtil {

    private val LOGGER: Logger = getLogger(LWJGLUtil::class)

    @JvmStatic
    fun log(msg: CharSequence) {
        LOGGER.info(msg.toString())
    }

    @JvmStatic
    fun log(msg: String?) {
        LOGGER.info(msg)
    }
}